hotel_reservation/static/src/js/hotel_room_summary.js

```diff
@@ -0,0 +1,106 @@
+odoo.define('hotel_reservation.hotel_room_summary', function (require) {
+'use strict';
+
+var core = require('web.core');
+var Widget= require('web.Widget');
+var widgetRegistry = require('web.widget_registry');
+var FieldManagerMixin = require('web.FieldManagerMixin');
+var time = require('web.time');
+var _t = core._t;
+var QWeb = core.qweb;
+var FormView = require('web.FormView');
+var rpc = require('web.rpc');
+
+
+var MyWidget = Widget.extend(FieldManagerMixin, {
+
+	template: "hotel_template",
+    view_type: "form",
+    init: function (parent, model, state) {
+        this._super(parent);
+        FieldManagerMixin.init.call(this);
+        // init code here
+        var id = 1;
+        rpc.query({
+            model: 'room.reservation.summary',
+            method: 'read',
+            args: [[id],['name'],['date_from']],
+        }).then(function(res) {
+            });
+        this.set({
+            date_to: false,
+            date_from: false,
+            summary_header: false,
+            room_summary: false,
+        });
+        this.summary_header = [];
+        this.room_summary = [];
+        this.on("_onFieldChanged:date_from", this, function() {
```

Comment:
this doesn't work for some reason - nothing is printed to console when changing date_from